src/content/blog/podman-and-docker-rootless.md

```diff
@@ -0,0 +1,593 @@
+   # View existing contexts
+   docker context ls
+
+   # Create rootless context
+   docker context create rootless \
+       --description "Rootless runtime socket" \
+       --docker host="unix://$XDG_RUNTIME_DIR/docker.sock"
+
+   # Switch to the context
+   docker context use rootless
+
+   # Verify it works
+   docker ps
+   ```
+
+9. Docker Rootless requires no-bind-mounts mode
+
+   Docker Rootless has a limitation with bind mounts that affects DDEV. You must enable [`no-bind-mounts` mode](https://docs.ddev.com/en/stable/users/configuration/config/#no_bind_mounts):
+
+   ```bash
+   ddev config global --no-bind-mounts=true
+   ```
+
+   **Why this is needed**:
+
+   Docker Rootless sets ownership for bind mounts to `root` inside containers. This is a known issue:
+   - [Mounting a volume with rootless always assigns ownership to root](https://github.com/moby/moby/issues/45919)
+   - [Add ability to mount volume as user other than root](https://github.com/moby/moby/issues/2259)
+
+   The `root` user inside the container maps to your host user, but many services will not run as root:
+   - nginx runs as root without problems
+   - MySQL/MariaDB need extra configuration
+   - Apache and PostgreSQL will not run as root
+
+   Podman Rootless fixes this with the `--userns=keep-id` option, which keeps user IDs the same. Docker Rootless does not have this option.
+
+   The `no-bind-mounts` mode fixes this by using Mutagen for the `web` container.
+
+## macOS
+
+macOS users can use Podman and Podman Desktop, but setup has its own challenges. Docker Rootless is not available on macOS.
+
+### Installing Podman
+
+Install Podman using Homebrew:
+
+```bash
+brew install podman
+```
+
+Or install [Podman Desktop](https://podman-desktop.io/docs/installation/macos-install) if you prefer a GUI.
+
+For more information, see the [official Podman installation guide for macOS](https://podman.io/docs/installation#macos) and [Podman tutorials](https://github.com/containers/podman/tree/main/docs/tutorials#readme).
+
+### Installing Docker CLI
+
+```bash
+brew install docker
+```
+
+### Configuring Podman
+
+After installing Podman, initialize and start the Podman machine:
+
+```bash
+podman machine init
+podman machine start
+```
+
+The Podman machine will automatically configure the socket. Verify it's running:
+
+```bash
+podman info --format '{{.Host.RemoteSocket.Path}}'
+```
+
+If you have Docker CLI installed, you can create a context for Podman:
+
+```bash
+# Create Podman context
+docker context create podman-rootless \
+    --description "Podman (rootless)" \
+    --docker host="unix://$(podman info --format '{{.Host.RemoteSocket.Path}}')"
+
+# Switch to the new context
+docker context use podman-rootless
+
+# Verify it works
+docker ps
+```
+
+Otherwise, set the `DOCKER_HOST` environment variable in your shell profile (`~/.zshrc` or `~/.bashrc`):
+
+```bash
+export DOCKER_HOST=unix://$(podman info --format '{{.Host.RemoteSocket.Path}}')
+```
```

Comment:
I didn't test macOS, hoping for possible improvements or tips from @rfay in this section.